main/src/ca/uwaterloo/flix/language/phase/Monomorph.scala

```diff
@@ -619,6 +619,9 @@ object Monomorph {
       case Expression.FixpointProject(_, _, _, _, _, loc) =>
         throw InternalCompilerException(s"Unexpected expression near: ${loc.format}.")
 
+      case Expression.Par(_, loc) =>
```

Comment:
This should be moved after Spawn or alternatively to the bottom.

Reply:
See https://github.com/flix/flix/pull/4327#discussion_r928801498